deviceinfo.js

```diff
@@ -87,6 +87,14 @@ const devicesWithNotch = [
     brand: 'Huawei',
     model: 'Mate 20 Pro',
   },
+  {
+    brand: 'Huawei',
+    model: 'P30 Lite',
+  },
+  {
+    brand: 'Huawei',
+    model: 'P30 Pro',
+  },
```

Comment:
Could you please move these additional lines to the bottom (keeping the whole array alphabetically sorted)

Reply:
Hi,  I added it after `Huawei`, it's already sorted alphabetically, right?

Reply:
I copied the array to javascript console and tried `.sort()` function and the array still the same.

Reply:
I do like that it is alphabetized but I'd be hard-pressed to try harder than actually testing sort() output :-)